js/server.js

```diff
@@ -38,7 +38,7 @@ var Server = function(config, callback) {
 			res.status(403).send("This device is not allowed to access your mirror. <br> Please check your config.js or config.js.sample to change this.");
 		});
 	});
-	app.use(helmet());
+	app.use(helmet(config.helmetOptions));
```

Comment:
@nhubbard helmet takes care of setting sane defaults for us, and I think allowing it to do so would be better since specifying the defaults in MM would be redundant. (Unless we want to override some of them by default in MM, but I don't think that is necessary.)

@cowboysdude The issue this PR solves is allowing MM to be shown in an iframe. My use for this feature is to show it on a Chromecast. It doesn't have any effect on MM modules using iframes.